flutter-studio/README.md

```diff
@@ -17,3 +17,40 @@ To set up a development environment:
 6. Select the `community-main` module and add module
    dependencies to `Dart-community`, `flutter-intellij-community`,
    and `flutter-studio`.
+
+The GUI tests must be run in the same version of IntelliJ used by
```

Comment:
We have some run configurations checked in already - one for unit tests, and a provisional one for integration tests. At some point, we should add a checked in run config for ui integration tests (and likely delete the old integration tests run config).